api/rosetta/errors.go

```diff
@@ -124,3 +124,14 @@ func UnknownBlock(fail failure.UnknownBlock) Error {
 		},
 	}
 }
+
+func UnknownTransaction(fail failure.UnknownTransaction) Error {
```

Comment:
This will likely change a bit with the new rosetta error PR incoming.. Whichever comes first :)